ocp_qp/d_ocp_qp_kkt.c

```diff
@@ -32,6 +32,7 @@
 #include <blasfeo_target.h>
 #include <blasfeo_common.h>
 #include <blasfeo_d_aux.h>
+#include <blasfeo_d_aux_ext_dep.h>
```

Comment:
why is this needed? there should be no printings of blasfeo stuff (otherwise please report here the issue)

Reply:
You're right, this is not necessary. I'll remove it